examples/chart/teleport-relay/values.yaml

```diff
@@ -0,0 +1,606 @@
+  # [`teleportClusterName`](#teleportClusterName).
+  tokenName: ""
+
+# teleportClusterName(string) -- is the name of the joined Teleport cluster.
+# Setting this value is required when joining via the
+# [Kubernetes JWKS or OIDC](../../reference/deployment/join-methods.mdx#kubernetes-jwks) join method.
+#
+# When this value is set, the chart mounts a kubernetes service account token
+# via a projected volume and configures Teleport to use it for joining.
+teleportClusterName: ""
+
+# joinTokenSecret -- manages the join token secret creation and its name. See
+# the [`joinParams`](#joinParams) section for more details.
+joinTokenSecret:
+  # joinTokenSecret.create(bool) -- controls whether the chart creates the
+  # Kubernetes `Secret` containing the Teleport join token. If false, you must
+  # create a Kubernetes Secret with the configured name in the Helm release
+  # namespace.
+  create: true
+  # joinTokenSecret.name(string) -- is the name of the Kubernetes Secret
+  # containing the Teleport join token used by the chart.
+  #
+  # If `joinTokenSecret.create` is `false`, the chart will not attempt to create
+  # the secret itself. Instead, it will read the value from an existing secret.
+  # `joinTokenSecret.name` configures the name of this secret. This allows you
+  # to configure this secret externally and avoid having a plaintext join token
+  # stored in your Teleport chart values.
+  #
+  # To create your own join token secret, you can use a command like this:
+  #
+  # ```code
+  # $ kubectl --namespace teleport create secret generic my-token-secret --from-literal=auth-token=<replace-with-actual-token>
+  # ```
+  #
+  # <Admonition type="note">
+  #
+  #   The key used for the auth token inside the secret must be `auth-token`, as
+  #   in the command above.
+  #
+  # </Admonition>
+  #
+  # ```yaml
+  # joinTokenSecret:
+  #   create: false
+  #   name: my-token-secret
+  #
+  # joinParams:
+  #   method: "token"
+  #   tokenName: ""
+  # ```
+  name: ""
+
+# proxyProtocol(bool) -- controls whether or not the connections coming from the
+# load balancer will include a PROXY protocol v2 header.
+proxyProtocol: false
+
+# log -- controls the agent logging.
+log:
+  # log.level(string) -- is the log level for the Teleport process.
+  # Available log levels are: `DEBUG`, `INFO`, `WARNING`, `ERROR`.
+  #
+  # The default is `INFO`, which is recommended in production.
+  # `DEBUG` is useful during first-time setup or to see more detailed logs for debugging.
+  level: INFO
+  # log.format(string) -- sets the log output format for the Teleport process.
+  # Possible values are `text` or `json`.
+  format: json
+
+# preStopDelay(string) -- the optional time that will pass between the pod
+# entering the Terminating state and the Teleport instance getting signaled to
+# begin its shutdown advertisement. Useful to allow load balancers to stop
+# routing connections to the terminating pod.
+preStopDelay: ""
```

Comment:
Should this be 30s by default? Let's assume users have no idea about the kube networking challenges and will not be able to understand/tune this value. We want the chart to work out of the box.